crates/cast/bin/args.rs

```diff
@@ -831,6 +831,48 @@ pub enum CastSubcommand {
         rpc: RpcOpts,
     },
 
+    /// Get the codehash for an account.
+    #[command(visible_alias = "ch")]
```

Comment:
imo codehash should be fine or chash

Reply:
Never seen `chash` so I think I'll leave it with no alias